### PR TITLE
Move scheduler read (get / batch get / scan) into coprocessor to separate read & write

### PR DESCRIPTION
**Pull request by breeswish**

### CODE REVIEW

**Comment by breeswish** [comment]
@zhangjinpeng1987 PTAL

File: src/coprocessor/endpoint.rs [hunk 1]
```diff
@@ -273,6 +278,9 @@ impl Display for Task {
 enum CopRequest {
     DAG(DAGRequest),
     Analyze(AnalyzeReq),
+    Get(GetRequest),
```
Comment (zhangjinpeng1987) [inline src/coprocessor/endpoint.rs]: A bit of ugly here. Get, BatchGet, Scan is not the same level with DAG.

Comment (siddontang) [reply src/coprocessor/endpoint.rs]: +1

Comment (zhangjinpeng1987) [reply src/coprocessor/endpoint.rs]: How about KV(KvRequest)?

Comment (breeswish) [reply src/coprocessor/endpoint.rs]: It is expected to be refined in a later version.

Comment (siddontang) [reply src/coprocessor/endpoint.rs]: Here I think you just need a read thread pool to handle all Scheduler and Coprocessor read, you can dispatch the Scheduler and Coprocessor in gRPC service directly.

**Pull request closed without merging**